src/components/pages/Navbar/Navbar.js

```diff
@@ -43,6 +45,21 @@ const Navbar = ({ isAuthenticated, userProfile, getProfile }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
We could open an issue here to set this useEffect to reach the DS API for the current user rather than the Postgres it is currently aimed at. Additionally there would be a backend ticket to go with it. That would also allow for the storage of the profile id as a variable to be used for the request object to send to the profile availability route rather than the current hard coded state. It may depend on the completion of the AuthO setup for the backend as well though, that I don't know.

Reply:
Yes I believe that Elijah has changes that still need to get patched through, to provide a working state of the application in the interim. We will see how things go. [BUG/restoring-access-to-data](https://github.com/BloomTech-Labs/underdog-devs-be-a/pull/149)

Reply:
Actually, this is one case where we'll want to keep using the endpoint described here exactly the same way we are now. What's happening here is that we're using `axiosWithAuth` to pass back the token received from either Okta or Auth0, decoding that in the BE, and searching for the user it describes in our DB. As we're keeping user data in the PostgreSQL database as well as in the Mongo database, this is completely fine.

Another note to make is the delineation of user info between the two databases:

**PostgreSQL**

- Okta/Auth0 ID
- Email Address
- Username
- First Name
- Last Name

**Mongo**

- Everything from the user's initial application
- The rest of the data we have our table on PostgreSQL storing can be shifted over to DS as well


This call is the initial grabbing of user info post login, feeding their info to the rest of the app - any other data we need we can call for elsewhere or through additional Redux actions like on line 41.

Reply:
Oh, and yes, this is hardcoded currently as a workaround for the lack of Auth - this will change soon!